### PR TITLE
documentation for values in guac.yml

### PR DESCRIPTION
**Pull request by iblameRishi**
Added documentation file for the values in guac.yml with these 2 under each value:
 - Description
 - When to change

### CODE REVIEW

File: guac-configuration.md
Comment (funnelfiasco) [inline guac-configuration.md:100]: Probably better to make this an h3, since the deps.dev certifier is a specific certifier

File: guac-configuration.md
Comment (funnelfiasco) [inline guac-configuration.md:116]: Let's use "Collector-Subscriber" throughout unless it's the name of a setting. This makes it a little less jargon-y.

**Review by funnelfiasco — CHANGES_REQUESTED** [review]
Thanks, @iblameRishi, this looks great! I have a couple of minor changes for you and some input needed from maintainers.

You will need to signoff on all of your commits before we can merge this. See https://docs.pi-hole.net/guides/github/how-to-signoff/ for instructions on how to that,

File: guac-configuration.md
Comment (pxp928) [reply guac-configuration.md:14]: I think we remove everything except for ENT as the others are not supported and the community is not using them.

Comment (pxp928) [reply guac-configuration.md:14]: There are a few ENT Specific ones as seen here: https://github.com/guacsec/guac/blob/main/container_files/ent/guac.yaml

```

# Ent config
db-driver: postgres
db-address: postgres://guac:guac@postgres:5432/guac?sslmode=disable
db-migrate: true
```

**Review by iblameRishi — COMMENTED** [review]
Added the changes, sorry about the delay but please let me know if anything else needs changing!

**Review by funnelfiasco — APPROVED** [review]
Looks great, thanks!

**Comment by funnelfiasco** [comment]
@iblameRishi two more things:

1. You need to signoff on your commits. See https://github.com/guacsec/guac-docs/pull/157/checks?check_run_id=31656079493 for instructions or message me in #guac on the OpenSSF Slack if you need help.
2. Looks like the CI run failed on the formatter. A few of your lines are probably longer than 80 characters. You can fix this with `npx --yes prettier --write --prose-wrap always *.md` and then committing the changes. (Feel free to squash all of your commits on this PR into one while you're doing this)

After that, it should be good to go.

**Comment by iblameRishi** [comment]
Added the signoffs and formatted the markdown file!

**Pull request merged**
